index.html

```diff
@@ -52,7 +52,13 @@
         style: 'text-align: right;',
       },
     };
-    (location.hash == "#/" || location.hash == "") ? location.hash = "#/README" : {};
+    const redirects = [
+      ["#/", "#/README"], 
+      ["", "#/README"], 
+      ["#/ModerationAPI/Against", "#/Archive/ModerationAPI/Against"],
+      ["#/GettingStarted", "#/GettingStarted/QuickStart"],
+    ];
+    redirects.forEach(r => location.hash == r[0] ? location.hash = r[1] : {});
```

Comment:
@rubybb There we go, thanks for feedback! I agree declaring an array looks cleaner. 🙂